editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/submissionDataTypesAndQueries.ts

```diff
@@ -0,0 +1,75 @@
+import gql from "graphql-tag";
+import { client } from "lib/graphql";
+
+type PaymentRequest = {
+  id: string;
+  createdAt: string;
+  paidAt: string;
+  govPaymentId: string;
+};
+
+type PaymentStatus = {
+  govPaymentId: string;
+  createdAt: string;
+  status: string;
+};
+
+type BopsApplication = {
+  id: string;
+  submittedAt: string;
+  destinationUrl: string;
+};
+
+type EmailApplication = {
+  id: string;
+  recipient: string;
+  submittedAt: string;
+};
+
+type UniformApplication = {
+  id: string;
+  submittedAt: string;
+};
+
+export type SubmissionData = {
+  sessionId: string;
+  submittedAt: Date | string;
+  paymentRequests: PaymentRequest[] | null;
+  paymentStatus: PaymentStatus[] | null;
+  bopsApplications: BopsApplication[] | null;
+  uniformApplications: UniformApplication[] | null;
+  emailApplications: EmailApplication[] | null;
+};
+
+export async function fetchSubmittedApplications(
+  flowSlug: string,
+  teamSlug: string,
+) {
+  const { data } = await client.query({
+    query: gql`
+      query SubmittedApplications($service_slug: String!, $team_slug: String!) {
+        submission_services_summary(
```

Comment:
```suggestion
        submissionServicesSummary: submission_services_summary(
```

Destructing to camelCase here means that we can avoid snake_case when handling the results in the React component 👌

Reply:
Great spot, also fixed here: cc2eb174e138a23046a710c38da83bdcc9d63a54